build_container.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
you might want to add a `set -e` here in order to abort if things fail (`-x` might also make sense for better logs)